Automattic/wp-calypso
# Added a toggle for marking membership subscriptions as donation.

Adds a new toggle option in the earn/payments-plans that allows the user to mark a subcription as a Donation. We didn't have this option before because all donation subscriptions were not displayed in the interface. With this change we will give the users the option to configure them.

#### Changes proposed in this Pull Request

* Add a new toggle in earn/payments-plans for marking a subscription as Donation.

#### Testing instructions



* Apply these diffs on your sandbox: `arc patch D77465 && arc patch D77864 --nobranch`
* Atomic: use Jetpack beta and switch to `add/membership-endpoint-updates` branch.
* Go to /earn/payments-plans/ and add a new payment plan.
* Press the Mark as a donation toggle.
* Refresh the list view page
* Make sure that the newly created donation is visible.
* After you test the Jetpack PR (where you added the donations block) make sure that the hidden donation subscriptions are not visible (the ones that are created by the Donation block.


Related to #61908
Printscreen:
![Screenshot 2022-05-19 at 13 25 57](https://user-images.githubusercontent.com/995955/169272646-bac18a74-bc3d-4356-baf3-02e041c4c75f.png)
